docs/installation.md

```diff
@@ -2,233 +2,287 @@
 
 ## Overview
 
-This document walks you through the installation of the Zonemaster *Backend* in the following sections:
+Zonemaster *Backend* needs to run on an operating system. One can choose any of
+the following OS to install the *Backend* after having the required
+[Prerequisites](#prerequisites).
 
-1. [Choices](#choices) lets you make all choices up front that this guide presumes you've made.
-2. [Prerequisites](#prerequisites) lets you check all assumptions made by this guide about pre-installed software.
-3. [Dependencies](#dependencies) shows you how to install all software that your Zonemaster *Backend* will depend on.
-4. [Installation](#installation) shows you how to install the *Backend* itself.
-5. [Configuration](#configuration) shows you how to configure the *Backend*.
-6. [Startup](#startup) shows you how to start the *Backend*.
-7. [Post-installation sanity check](#post-installation-sanity-check) lets you verify that your *Backend* has started correctly.
+* [CentOS](#centos 7)  
+* [Debian](#debian 8 (Jessie))
+* [Ubuntu](#ubuntu 16.04)
+* [FreeBSD](#freebsd) 

```

Comment:
For the numbers in the headlines, i followed the same process as in the install docs of Engine and CLI. While testing, i tried different ways of numbering, and finally, i felt this way of numbering more easier, since i know, that to install the backend, i have to do '8' steps.